tests/bson/bson-objectid_error-001.phpt

```diff
@@ -7,10 +7,11 @@ BSON BSON\ObjectID #001 error
 <?php
 require_once __DIR__ . "/../utils/basic.inc";
 
-$classname = BSON_NAMESPACE . "\\ObjectID";
-throws(function() use($classname) {
-$id = new $classname(new stdclass);
+throws(function() {
+    $classname = BSON_NAMESPACE . "\\ObjectID";
+    new $classname(new stdclass);
```

Comment:
There was already a space in the old code, so that's why indentation looks off. It's still 4 spaces, though.